packages/react-chat/src/assets/svg/sound-off.svg

```diff
@@ -1,9 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<svg viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg"
-  xmlns:xlink="http://www.w3.org/1999/xlink">
-  <g transform="translate(0.000000, 1.000000)">
-    <path
-      d="M7.7752,1.68539637 C7.8064,2.72139637 7.9424,4.88299637 7.9424,6.70939637 C7.9424,8.53419637 7.8064,10.6965964 7.7752,11.7325964 C7.752,12.5109964 7.344,12.9493964 6.5056,12.3117964 C5.2616,11.3645964 4.0064,10.4333964 3.548,10.0933964 C3.0904,9.75259637 2.6424,9.37819637 1.6088,9.37819637 C0.576,9.37819637 0.1568,8.73979637 0.0952,8.15579637 C0.0387333333,7.62119637 0.00579444444,6.94475748 0.000696759259,6.75007785 L0,6.70939637 C0,6.59739637 0.0336,5.84539637 0.0952,5.26219637 C0.1568,4.67819637 0.576,4.04059637 1.6088,4.04059637 C2.6424,4.04059637 3.0904,3.66539637 3.548,3.32459637 C4.0064,2.98379637 5.2616,2.05339637 6.5056,1.10619637 C7.344,0.468596368 7.752,0.906996368 7.7752,1.68539637 Z M15.8333827,4.16661726 C16.0384679,4.37170241 16.0564212,4.69824329 15.8750556,4.92457565 L14.5812728,6.53913099 C14.3654421,6.80847342 14.3654421,7.19152658 14.5812728,7.46086901 L15.8750556,9.07542435 C16.0564212,9.30175671 16.0384679,9.62829759 15.8333827,9.83338274 C15.6282976,10.0384679 15.3017567,10.0564212 15.0754243,9.87505557 L13.460869,8.58127282 C13.1915266,8.36544212 12.8084734,8.36544212 12.539131,8.58127282 L10.9245757,9.87505557 C10.6982433,10.0564212 10.3717024,10.0384679 10.1666173,9.83338274 C9.9615321,9.62829759 9.94357876,9.30175671 10.1249444,9.07542435 L11.4187272,7.46086901 C11.6345579,7.19152658 11.6345579,6.80847342 11.4187272,6.53913099 L10.1249444,4.92457565 C9.94357876,4.69824329 9.9615321,4.37170241 10.1666173,4.16661726 C10.3717024,3.9615321 10.6982433,3.94357876 10.9245757,4.12494443 L12.539131,5.41872718 C12.8084734,5.63455788 13.1915266,5.63455788 13.460869,5.41872718 L15.0754243,4.12494443 C15.3017567,3.94357876 15.6282976,3.9615321 15.8333827,4.16661726 Z"
-      fill="currentColor"></path>
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
should we add width and height attrs? Because we have it in our icon lib.

Reply:
in this repo we don't have size attributes in the icons to control size via props